logprep/runner.py

```diff
@@ -210,18 +210,37 @@ def reload_configuration(self):
         """
         if self._configuration is None:
             raise CannotReloadWhenConfigIsUnsetError
-
-        new_configuration = Configuration.create_from_yaml(self._yaml_path)
-
+        try:
+            new_configuration = Configuration.create_from_yaml(self._yaml_path)
+        except (requests.RequestException) as error:
```

Comment:
What about `No such file or directory` error? If I run the quickstart config, not with a SimpleHTTPServer but really just the plain old config file I get the error `No such file or directory` if the original config file should get lost before the next reload attempt. Logprep then goes into a state of shutting down, but not doing it completely.

Reply:
as you command... -> DONE :-D